babel/dates.py

```diff
@@ -437,7 +454,10 @@ def get_timezone_location(dt_or_tzinfo=None, locale=LC_TIME):
                          the timezone; if `None`, the current date and time in
                          UTC is assumed
     :param locale: the `Locale` object, or a locale string
-    :return: the localized timezone name using location format
+    :param return_city: True or False
```

Comment:
I did added that info but at wrong location, Should I move it up?
 ` :param return_city: True or False`
   `:return: the localized timezone name using location format;`
   `          if return_city is True then return
            The exemplar city (location) for the time zone`